python/lsst/daf/butler/registry/dimensions/static.py

```diff
@@ -265,15 +271,22 @@ def sync(self, record: DimensionRecord, update: bool = False) -> bool | dict[str
                     # Inserted a new row, so we just need to insert new
                     # overlap rows (if there are any).
                     self._insert_overlaps(
-                        record.definition, db_rows.overlap_insert_rows, overlap_delete_rows=[]
+                        record.definition,
+                        db_rows.overlap_insert_rows,
+                        overlap_delete_rows=[],
```

Comment:
A few of these files seem to have a bunch of places where unchanged code was randomly reformatted... not sure if there is a mismatched `black` version in play or what.

Reply:
Thanks.  I did have a misconfigured editor early on in my work on this branch and I had thought I fixed all of the fallout, but it seems I did miss some.